src/quaternion.rs

```diff
@@ -474,7 +474,7 @@ impl<S: BaseFloat> From<Quaternion<S>> for Basis3<S> {
 
 impl<S: BaseFloat> Rotation<Point3<S>> for Quaternion<S> {
     #[inline]
-    fn look_at(dir: Vector3<S>, up: Vector3<S>) -> Quaternion<S> {
```

Comment:
There should be `{lh|rh}` variants.

Reply:
I was hoping to consolidate the look functions in `Transform` and `Rotation` in a separate PR since it is more involved. This looks like it is at least has consistent naming. What do you think?

Reply:
I guess my thinking is that it needs the handedness for it to be consistent and I don't think the implementations are consistent right now (i.e. they're not all LH or all RH and some of the implementations are on the raw structs instead of the traits).  

What I think needs to happen is that someone needs to go through the remaining `look_` function and determine if it's currently `lh` or `rh` and deprecate it in favor of an explicitly named version of that function. While checking these, the corresponding missing `lh` or `rh` version can also be added. I think you've done this for `Matrix3`, but the other ones are missing the handedness suffix. I skipped over the ones I wasn't sure of rather than just renaming them to `look_to` (without the suffix) because they'll just need to be renamed again. 

I was also thinking that it doesn't really make sense to have `look_at_` variants for structures that can only represent rotation. I think these should **only** have `look_to_` functions. We should probably sort out these kinds of questions before considering the removal of `Rotation`.

Reply:
> We should probably sort out these kinds of questions before considering the removal of Rotation.
Agreed, so shall we punt that to another PR?
For this PR I am happy with having the look_at vs look_to consistency throughout cgmath.